src/libs/location/src/Character.h

```diff
@@ -251,6 +251,15 @@ class Character : public Entity
     //--------------------------------------------------------------------------------------------
     // Character animation
     //--------------------------------------------------------------------------------------------
+  protected:
+    // Procedural head look
+    enum HeadLookState
```

Comment:
Could you use `enum class` instead, please? There is a lot of nasty legacy things, but for the new code, we prefer to avoid bad practices.